app/src/config/appConfig.ts

```diff
@@ -17,3 +17,4 @@ if (process.env.NODE_ENV == "production") {
 }
 
 export default appConfig;
+export const githubAuthEndpoint = (config: AppConfig) => `${config.apiUrl()}/oauth2/authorization/github`;
```

Comment:
I added this util when I thought we might have to make a special case to exclude this url from login redirection, so would need to build from separate places. Didn'tdo that in the end, but it's probably ok to leave this as the responsibility of appConfig module.